ansible_3par_docker_plugin/uninstall/uninstall_hpe_3par_volume_driver.yml

```diff
@@ -15,12 +15,14 @@
       include: remove_multipath.yml
 
 - name: Remove HPE 3PAR Volume Driver for Kubernetes/OpenShift
-  hosts: masters
+  hosts: "{{groups['masters'][0]}}"
   become: root
 
+
   tasks:
     - name: Start Dynamic Provisioner (doryd) on Master node
```

Comment:
Task name should be "Stop Dynamic Provisioner (doryd) on Master node" instead of Start.

Reply:
Done